simplerunner.lua

```diff
@@ -551,6 +563,32 @@ function runner.parse_commandline(args)
         end
     end
 
+    if g_opt.optionsfile then
+        local success, options = loadjson(g_opt.optionsfile)
+        if not success then
+            print("Error "..options)
+            os.exit(1)
+        end
+
+        local file_benchmarks, file_excludes = options.benchmarks, options.excludes
+        options.benchmarks = nil
+        options.excludes = nil
+
+        -- If the optional file doesn't specify a list of benchmarks allow the command line options set them
+        if #file_benchmarks > 0 then
+            g_opt.benchmarks = file_benchmarks
+        end
+
+        if #file_excludes > 0 then
+            g_opt.excludes = file_excludes
+        end
+
+        -- Merge options from the file with command line. Conflicting command line options will get overwritten for simplicity.
```

Comment:
I don't understand the first sentence.

Reply:
@ltratt how about "Merge values from the option file with command line options. Conflicting command line options will get overwritten for simplicity."

Reply:
Ok, now I get it. But I think conflicting options should probably lead to an error, otherwise people are going to do all sorts of weird things. Thoughts?

Reply:
@ltratt I wanted to have like an overlay system where the base options could be set by an option file but the user could override some value on the command line. 
I started out with the commandline overriding the options file but i couldn't tell if a command line option was set because there are default values in the optional table already but i think i can remove them and go back to command line always overriding

Reply:
I must admit that I think this is too much complexity, at least until we have a concrete use case. Can we simplify this stuff please?

Reply:
The use case was the same as krun overlay files to quickly experiment from a base setup and to customize to an particular environment but did simplify from what i started